### DISCUSSION TITLE
BananaPi-Zero instead of RaspberryPi Zero?

### DISCUSSION THREAD

**Discussion by RetroWizzard** [frntc/RAD#20] [Q&A] [answered]
The RAD Zero works fine with my RaspberryPi Zero 2 W.
However since the RPi requires a second mortgage on the house these days, I had the crazy idea to use a BananaPi-Zero instead.
Unfortunately it doesn't work just out of the box.
Do you have any ideas suggestions what could make it work or where to check?

Thanks a lot.

**Comment by frntc** [comment, marked as answer]
The Banana-Pi is built around a different SoC than the RPi Zero, and the RAD uses the Circle library for bare metal programming which is made for RPis only. Without replacing all low-level code it won't work. I also have no experience how fast or slow the GPIOs of the Bananas are, i.e. if the SoC is suitable.

**Reply by RetroWizzard** [reply]
Ah, I see. Thanks for clarifying.

**Comment by ZaPpi3** [comment]
I use https://rpilocator.com/ to check availability.
The Pi Zero W seems to be in good supply these days at retail level.
Pimoroni even lets you purchase 10 at a time for the first time ever, so I imagine they have become less of an issue to find recently.

**Reply by frntc** [reply]
You need a Pi Zero _2_! A "Zero 1" will not work (again, different/old SoC).

**Reply by ZaPpi3** [reply]
Yeah, my bad. I use the 3A, so I didnt notice. 
The site was the important bit of my post though. It really is pretty fast at getting stocked retailers.
